fews_3di/utils.py

```diff
@@ -121,7 +124,11 @@ def _read_property(self, property_name, optional=False):
 
         elif property_name == "saved_state_expiry_days":
             value = int(string_value)
-
+        elif property_name == "api_host":
+            # check if a different api_host is provided in settings.xml
+            # otherwise use the default provided with setattr
+            if string_value:
+                value = string_value
```

Comment:
These lines aren't needed: the default "else" is to use the value as a string. The "if/elif" is for convertion to integers and so.

```suggestion
```